web/styles/settings.css

```diff
@@ -639,6 +639,11 @@ input[type="checkbox"] {
     }
 }
 
+.zulip-icon-at-sign {
+    position: relative;
+    top: 3px;
```

Comment:
I'm not a fan of these kinds of fiddly, position-based adjustments. It would be useful to look into making a parent container around the icon and label, and using a modern technique like flexbox to get the alignment to work.

Reply:
I've update the PR with this changes

- Add a div container inside th cell with a class of notification-table-header-content.
- Set baseline alignment on the group, and center alignment on the icons on a per-flex-item basis.
- Added a new container for at-sign and all text to center them vertically;